### PR TITLE
fixed two bugs

### PR DESCRIPTION
**Pull request by radomir9720**
1. bug with portrait images aspectRatio(crop grid is out of boundaries). In the current version crop grid is painted relative to horizontal edges, but vertical(top edge and bottom edge) are not taken into account. This means that crop grid's width is always equal to device screen's width, what, of course, is wrong. This was fixed by checking does crop grid go beyond  image boundaries. If so, then we change crop grid width to corresponding to crop grid height. 
2. bug with identification of scaling and moving(dragging) gestures. In the current version for scaling you need to touch with two fingers exactly at the same time, what is pretty hard, to be honest. This was fixed by counting pointers(user fingers on the screen).
3. Syntax fixes

### CODE REVIEW

**Comment by lykhonis** [comment]
This is really great. I cannot validate but based on details it seems to fix the issues. Can you double confirm that you have not seen regressions of any sort after these changes as they touch quite a lot?

Thank you, I will merge and do update to newer version.

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -9,11 +9,7 @@
 /* Begin PBXBuildFile section */
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
 		3B3967161E833CAA004F5970 /* AppFrameworkInfo.plist in Resources */ = {isa = PBXBuildFile; fileRef = 3B3967151E833CAA004F5970 /* AppFrameworkInfo.plist */; };
-		3B80C3941E831B6300D905FE /* App.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3B80C3931E831B6300D905FE /* App.framework */; };
```
Comment (lykhonis) [inline example/ios/Runner.xcodeproj/project.pbxproj:12]: Are these changes accidental?

Comment (radomir9720) [reply example/ios/Runner.xcodeproj/project.pbxproj:12]: Yes. I think it was generated after build project.

Comment (lykhonis) [reply example/ios/Runner.xcodeproj/project.pbxproj:12]: Could you please clean those up to avoid side effects if any :/

Comment (lykhonis) [reply example/ios/Runner.xcodeproj/project.pbxproj:12]: Nvm. I realized this is example :)

**Comment by radomir9720** [comment]
> This is really great. I cannot validate but based on details it seems to fix the issues. Can you double confirm that you have not seen regressions of any sort after these changes as they touch quite a lot?
> 
> Thank you, I will merge and do update to newer version.

Yes, i didn't see regressions.

**Pull request merged**